doc/Logging.md

```diff
@@ -0,0 +1,16 @@
+How does Happy test log work
+============================
+
+## Happy produced logs
+  * Happy host logs
```

Comment:
Let's describe what types of logs are produced, specifically.  What's the difference between host and device?

Reply:
done